Source/CarthageKit/Project.swift

```diff
@@ -1378,6 +1357,104 @@ internal func frameworksInDirectory(_ directoryURL: URL) -> SignalProducer<URL,
 	}
 }
 
+internal func copyFrameworksAndCreateVersionFile(
+	_ frameworkURLs: [URL],
+	from directoryURL: URL,
+	to destinationURL: URL,
+	projectName: String,
+	pinnedVersion: PinnedVersion,
+	toolchain: String?
+) -> SignalProducer<URL, CarthageError> {
+	return SignalProducer(value: frameworkURLs)
+		.flatten()
```

Comment:
`SignalProducer(value: frameworkURLs).flatten()` can be written as `SignalProducer(frameworkURLs)`

Reply:
Simplified, I'll push sooner